randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -902,6 +923,7 @@ Extra - room_id: [24]
   * Extra - door_idx: (80,)
   > Pickup (Power Bomb Tank)
       All of the following:
+          # Hi Jump not required, with area effect/wall cutting weapon (wave, diffusion, power bomb) or bombs
```

Comment:
Is the bomb option for a "jump bombjump"?

Reply:
For destroying the other blocks in the sequence if you don't have something that goes through the wall, but you probably could trivially shoot them by jumping from the ledge grab - can test that